apis/vmware/v1beta1/vspheremachine_types.go

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 package v1beta1
 
 import (
+	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha1"
```

Comment:
I think we also should inline the type here

Reply:
We probably want to avoid dependencies between our two api packages. 

I would prefer inlining it in the vmware/v1beta1 package as well.

But let's see what @randomvariable thinks

Reply:
Just to avoid the case when we forgot to keep them in sync. It's a simple change and I'm fine with both.

Reply:
For me it's mainly about not having dependencies between the both API packages and being able to evolve them independently if we have to

Reply:
Could be also relevant for folks importing only the supervisor Go types. As with this dependency they also get all the transitive dependencies of apis/v1beta1.

Another problem is if we want to evolve apis/v1beta1 and vmware/v1beta1 independently. E.g. if we bump apis/v1beta1 to apis/v1beta2 do we have to bump vmware/v1beta1 as well

Reply:
In that case using a separate type (even with the same name) seems better.